src/app/(post-login)/submission/program-table/Countries.tsx

```diff
@@ -18,14 +18,17 @@
  */
 'use client';
 
-const Countries = ({ countries }: { countries: string[] }) => (
+import { Maybe } from '@/__generated__/graphql';
+
+const Countries = ({ countries }: { countries: Maybe<Maybe<string>[]> | undefined }) => (
 	<div>
-		{countries.map((country, i) => (
-			<div key={country}>
-				{country}
-				{i < countries.length - 1 && ','}
-			</div>
-		))}
+		{countries &&
+			countries.map((country, i) => (
```

Comment:
If you're trying to account for countries being defined, this is a cleaner approach since ES2020.
and in this case, the Optional Chaining operator also checks that map is a method of countries, for good measure...
hard to tell with that nested Maybe whether it'd be useful or even necessary.
```suggestion
		{countries?.map?.((country, i) => (
```